elfpy/data/postgres.py

```diff
@@ -59,6 +59,24 @@ def build_postgres_config() -> PostgresConfig:
     return PostgresConfig(**arg_dict)
 
 
+def query_tables(session):
+    """Return a list of tables in the database."""
+    stmt = text(
```

Comment:
I saw this in another PR and left a comment about doing this in a sqlAlchemy kinda way.  Same here.